jps-plugin/src/org/intellij/erlang/jps/builder/ErlangFileDescriptor.java

```diff
@@ -22,13 +22,24 @@
 import com.intellij.util.xmlb.annotations.Attribute;
 import com.intellij.util.xmlb.annotations.Tag;
 
-import java.util.List;
+import java.util.Collection;
 
 @Tag("module")
 public class ErlangFileDescriptor {
   @Attribute("path")
   public String myErlangModulePath;
+
   @Tag("dependencies")
   @AbstractCollection(surroundWithTag = false, elementTag = "dependency")
-  public List<String> myDependencies = ContainerUtil.newArrayList();
+  public Collection<String> myDependencies;
```

Comment:
It's a bad idea to use a Collection for persistence. You don't know what type of collection are you'll get after it's deserialized from xml, hence you don't know what the iteration order is.

What was wrong with the List, anyway?